po/he.po

```diff
@@ -1,14 +1,14 @@
-# SOME DESCRIPTIVE TITLE.
+# DARKTABLE HEBREW TRANSLATION.
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
-# This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# This file is distributed under the same license as Darktable.
```

Comment:
it's only `darktable`.
not `Darktable`, not `DarkTable`, not `Dark Table`, etc.